rosidl_typesupport_opensplice_cpp/src/identifier.cpp

```diff
@@ -18,6 +18,6 @@ namespace rosidl_typesupport_opensplice_cpp
 {
 
 ROSIDL_TYPESUPPORT_OPENSPLICE_CPP_EXPORT
-const char * typesupport_opensplice_identifier = "opensplice_static";
+const char * typesupport_identifier = "opensplice_static";
```

Comment:
The PR doesn't change the value of the identifier. It only aims to unify the naming of the symbols.